lib/modules/datasource/metadata-manual.ts

```diff
@@ -54,6 +54,10 @@ export const manualChangelogUrls: Record<string, Record<string, string>> = {
     neo4j: 'https://neo4j.com/release-notes/',
     'whitesource/renovate': 'https://github.com/whitesource/renovate-on-prem',
   },
+  maven: {
+    'com.figure.gradle.semver-plugin': 
```

Comment:
this seems to not enough, the group prefix is missing.

checkout the packagefiles with updates debug log message. there you should find the full package name which is required here.

Reply:
@viceice sorry for the super late response here, got a bit distracted and finally turned on debug logging. I found the packagefiles debug log and have updated this PR to match!

```
{"depType":"plugin","depName":"com.figure.gradle.semver-plugin","packageName":"com.figure.gradle.semver-plugin:com.figure.gradle.semver-plugin.gradle.plugin","currentValue":"1.4.1"...
```

Let me know if there's any other changes you'd like here!